modules/apps/27-interchain-accounts/types/packet.go

```diff
@@ -1,9 +1,40 @@
 package types
 
 import (
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
 )
 
-func (packet InterchainAccountPacketData) GetBytes() []byte {
-	return sdk.MustSortJSON(ModuleCdc.MustMarshalJSON(&packet))
+// ValidateBasic performs basic validation of the interchain account packet data.
+// The memo may be empty.
```

Comment:
Should we enforce a max memo length?

Reply:
Sounds like a good idea! What were you thinking, something like 256 chars?

Reply:
Hmm, no idea. I'll do 256 for now and open an issue on cosmos/ibc